templates/workbooks/workbook_plot.html

```diff
@@ -40,7 +40,7 @@
         </div>
 
         <!-- Settings Flyout -->
-        <div class="fly-out settings-flyout plot-config" style="overflow-y: scroll;" id="plot-{{ worksheet.id }}-settings">
+        <div class="fly-out settings-flyout plot-config" style="overflow-y: auto;" id="plot-{{ worksheet.id }}-settings">
```

Comment:
Definitely--there's actually a lot of in-line styles running around, should I make a ticket just for those and then get them all folded in?